README.md

```diff
@@ -27,14 +27,19 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+`DOM is the interface which lets us grab and manipulate elements on a webpage. DOM translates elements in a webpage into objects. These objects can then be subsequently accessed using JavaScript. Thus, DOM acts a bridge or a conduit between the elements on the webpage and the programs written to access/manipulate those elements.`
```

Comment:
Pretty close! The DOM **is** the object representation of the HTML elements on a webpage. The browser engine inside your browser creates the DOM.